libraries/RC_Channel/RC_Channel.h

```diff
@@ -182,6 +182,7 @@ class RC_Channel {
         Q_ASSIST =            82, // disable, enable and force Q assist
         ZIGZAG_Auto =         83, // zigzag auto switch
         AIRMODE =             84, // enable / disable airmode for copter
+        CROW_SELECT =         86, // select CROW mode for diff spoilers;high disables,mid forces progressive
```

Comment:
Skipped one?

Reply:
hoping my Terrain sw goes in first...its 85..I'm lazy